anago

```diff
@@ -970,7 +970,10 @@ announce () {
     logecho "$WARNING: Email cannot be sent from GCB.  Execute the following" \
             "after a completed release to send email notification:"
     logecho
-    logecho "$ release-notify $RELEASE_VERSION_PRIME"
+    local nomock_flag
```

Comment:
nit, can you put put the declaration up above?  To follow the style of the repo.

Reply:
fixed.